schemas/context/optinout.schema.json

```diff
@@ -109,5 +109,5 @@
       "$ref": "#/definitions/optinout"
     }
   ],
-  "meta:status": "experimental"
+  "meta:status": "stabilizing"
```

Comment:
👍

Reply:
👎 - I think we need a larger conversation here about how "preference" objects should be handled.